### PR TITLE
task #3517 from PR #379 add ssh keys layout

### PR DESCRIPTION
**Pull request by tiwariav**
Styles issues fixed

### CODE REVIEW

**Comment by verysanghee** [comment]
tested, the translation seems to be mixed.

**Comment by verysanghee** [comment]
tested again, now we're good.

File: datacenterlight/templatetags/custom_tags.py [hunk 1]
```diff
@@ -21,7 +21,6 @@ def change_lang(context, lang=None, *args, **kwargs):
 
     return "%s" % url
```
Comment (pcoder) [inline datacenterlight/templatetags/custom_tags.py]: By deleting this line, I think we are breaking PEP8. Here's what flake8 complains
```
./datacenterlight/templatetags/custom_tags.py:24:1: E302 expected 2 blank lines, found 1```

Comment (tiwariav) [inline datacenterlight/templatetags/custom_tags.py]: wil fix in PR #428

File: datacenterlight/views.py [hunk 2]
```diff
@@ -19,7 +19,7 @@
 from utils.stripe_utils import StripeUtils
 from datetime import datetime
 from membership.models import CustomUser, StripeCustomer
-
+from oca.pool import WrongIdError
```
Comment (pcoder) [inline datacenterlight/views.py]: I think this import (WrongIdError) is not being used anywhere, so can be removed.

File: datacenterlight/views.py [hunk 3]
```diff
@@ -416,6 +419,7 @@ class OrderConfirmationView(DetailView):
     context_object_name = "order"
     model = HostingOrder
 
+
```
Comment (pcoder) [inline datacenterlight/views.py]: flake8 is complaining about too many blank lines.

./datacenterlight/templatetags/custom_tags.py:24:1: E302 expected 2 blank lines, found 1

**Comment by tiwariav** [comment]
closing this PR as it is merged with PR #428

File: datacenterlight/views.py [hunk 4]
```diff
@@ -208,6 +209,7 @@ def validate_storage(self, value):
 
     @cache_control(no_cache=True, must_revalidate=True, no_store=True)
     def get(self, request, *args, **kwargs):
+
```
Comment (pcoder) [inline datacenterlight/views.py]: Not sure why we would need this new line? The same for the following diff line.

**Comment by tiwariav** [comment]
closing this PR as its merged with PR #428

File: dynamicweb/settings/prod.py [hunk 5]
```diff
@@ -7,6 +7,7 @@
 
 EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
 
+
```
Comment (pcoder) [inline dynamicweb/settings/prod.py]: Not sure why we need this additional line

File: hosting/forms.py [hunk 6]
```diff
@@ -20,9 +26,11 @@ def clean(self):
         password = self.cleaned_data.get('password')
         is_auth = authenticate(email=email, password=password)
         if not is_auth:
-            raise forms.ValidationError("Your username and/or password were incorrect.")
+            raise forms.ValidationError(
```
Comment (pcoder) [inline hosting/forms.py]: We could add a translation for the text.

File: dynamicweb/settings/prod.py (hunk 5 above)
Comment (tiwariav) [inline dynamicweb/settings/prod.py]: fixed.

File: hosting/forms.py (hunk 6 above)
Comment (tiwariav) [inline hosting/forms.py]: translation added

**Pull request closed without merging**